src/screens/Home/Home.tsx

```diff
@@ -37,6 +38,70 @@ interface HomeProps {
 
 function Home(props: HomeProps) {
   const { classes, lastUpdated, isOnline, currentSite } = props;
+  useEffect(() => {
+    proccessData();
+  }, []);
+
+  const [outStandingPayments, setOutstandingPayments] = useState<number>();
+  const [toCharge, setToCharges] = useState<number>();
+  const [customerDetails, setCustomerDetails] = useState<number>();
+  const [unpaidReports, setUnpaidReports] = useState<number>();
+
+
+
+  const proccessData = () => {
+    calculateCustomerData(currentSite);
+    calculateUnpaidReports(currentSite)
+  }
+
+  const calculateCustomerData = (ss: SiteRecord) => {
+    // customers to charge:  haven't charged yet/ no meterreading
+    // outstanding payments: done the meter reading / charged, haven't paid yet
+    let toCharge = 0;
+    let outstanding = 0;
```

Comment:
`outstandingPayments` is a better variable name. Outstanding is weird because it's an actual word with a main definition that isn't relevant to what it's doing